lawnchair/src/app/lawnchair/ui/preferences/about/GithubService.kt

```diff
@@ -27,6 +27,20 @@ interface GitHubService {
         @Path("owner") owner: String,
         @Path("repo") repo: String,
     ): List<GitHubEvent>
+    
+    @GET("repos/{owner}/{repo}/commits")
+    suspend fun getCommits(
```

Comment:
Consistency with above code

```suggestion
    suspend fun getRepositoryCommits(
```